### ISSUE TITLE
Example doesn't compile

### ISSUE THREAD

**Issue by eaoliver** [Neo-Ciber94/next-server-task#2]
Steps to reproduce from: /next-server-task/examples/openai-image-generator

```
pnpm install 
pnpm build 
```

Module not found: Can't resolve 'next-server-task'
> 1 | import { TaskError } from "next-server-task";
  2 | import { createTask } from "next-server-task/server";

Module not found: Can't resolve 'next-server-task/server'
  1 | import { TaskError } from "next-server-task";
> 2 | import { createTask } from "next-server-task/server";
  3 | import { OpenAI } from "openai";

**Comment by 2-fly-4-ai** [comment]
Yeah I have the same issue.

**Comment by Neo-Ciber94** [comment]
I think the cause is because running `pnpm install` on the example do not build the library.

Try:

> Build the library

`cd package && pnpm build`

And then in other shell
`cd examples/openai-image-generator && pnpm install && pnpm build`

**Comment by 2-fly-4-ai** [comment]
Yeah, the typings and stuff are all off as well. But.. I got it to work, and I'm bulk processing about 15 images in 20sec+-. Nice job, thanks. It would be cool if we could track down the source of the issue. I'll just have to bypass it for now. 

This is a really nice solution to dealing with long-running serverless tasks.

**Comment by Neo-Ciber94** [comment]
@2-fly-4-ai Are you using the library from npm `npm i next-server-task` or from the source?

If you are using it directly from the source, I think may be related to not building the lib as I mentioned above.

**Comment by 2-fly-4-ai** [comment]
I think we're all installing it through npm. I'm getting red lines on the imports.
